internal/repository/prisma/dbsqlc/workflow_runs.sql

```diff
@@ -99,53 +99,53 @@ LIMIT
     COALESCE(sqlc.narg('limit'), 50);
 
 -- name: PopWorkflowRunsRoundRobin :many
-WITH running_count AS (
-    SELECT
-        COUNT(*) AS "count"
-    FROM
-        "WorkflowRun" r1
-    JOIN
-        "WorkflowVersion" workflowVersion ON r1."workflowVersionId" = workflowVersion."id"
-    WHERE
-        r1."tenantId" = $1 AND
-        r1."status" = 'RUNNING' AND
-        workflowVersion."id" = $2
-), queued_row_numbers AS (
+WITH workflow_runs AS (
     SELECT
         r2.id,
+        r2."status",
         row_number() OVER (PARTITION BY r2."concurrencyGroupId" ORDER BY r2."createdAt") AS rn,
         row_number() over (order by r2."createdAt" ASC) as seqnum
     FROM
         "WorkflowRun" r2
     LEFT JOIN
         "WorkflowVersion" workflowVersion ON r2."workflowVersionId" = workflowVersion."id"
     WHERE
-        r2."tenantId" = $1 AND
-        r2."status" = 'QUEUED' AND
-        workflowVersion."id" = $2
+        r2."tenantId" = @tenantId::uuid AND
+        (r2."status" = 'QUEUED' OR r2."status" = 'RUNNING') AND
```

Comment:
We are filling all available slots with a `QUEUED` or `RUNNING` status. We are obviously not going to pop the running status, but we want to figure out how many slots we have and only return queued runs in those slots.